src/lib/dataEntities/networks.ts

```diff
@@ -28,14 +28,16 @@ export interface L1Network extends Network {
 }
 
 export interface L2Network extends Network {
+  partnerChainIDs: number[]
   tokenBridge: TokenBridge
   ethBridge: EthBridge
   partnerChainID: number
-  isArbitrum: true
+  isArbitrum: boolean
   confirmPeriodBlocks: number
   retryableLifetimeSeconds: number
   nitroGenesisBlock: number
   nitroGenesisL1Block: number
+  isL3?: boolean
```

Comment:
Do we need this property? Can we compute that something is an Orbit chain like this:

```
network.isArbitrum && network.partnerChainIDs.length > 0
```

Reply:
we can do that

Reply:
what if `network` is an `L2Network`, and we want to check if `network` is L3? then we need to do something like:

```
await getL2Network(network.parentChainID).parentChainIDs.length > 0
```

it seems a bit complicated: to figure out if the chain of interest is an L3 we'd have to get its parent chain first.